rubocop/cop/i18n/deprecated_helper.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module I18n
+      # Checks for uses of the deprecated helper function, `ts`.
+      # Strings passed to it cannot be translated, and all calls
+      # will need to be replaced with I18n.t to enable UI translations
+      # in the future.
+      #
+      # @example
+      #  # bad
+      #  ts("This will only be in English :(")
+      #  ts("Hello %{name}", name: "world")
+      #
+      # @example
+      #  # good
+      #  t(".relative.path.to.translation")
+      #  t(".greeting", name: "world")
+      class DeprecatedHelper < RuboCop::Cop::Base
+        MSG = "Prefer Rails built-in `I18n.t` function over `ts`: the latter is not actually translatable. For more information, refer to https://github.com/otwcode/otwarchive/wiki/Internationalization-(i18n)-Standards."
```

Comment:
My initial reasoning behind the full `I18n.t` was that looking up just `t` for someone unfamiliar is nigh impossible. But I think with the examples in the now-linked "Internationalization Standards" page, that issue becomes a bit less problematic. Do you think it would be a good idea to include the a link to the Rails i18n docs on that page for completeness?

Reply:
Link to the i18n guide on the wiki page? May be good, may be confusing because the guide walks through a setup that is already done here (and different). The page already links to subsections like the explanation on lazy lookup, but it is a bit hidden in the bullet points. So I could see a link to the https://guides.rubyonrails.org/i18n.html#overview-of-the-i18n-api-features being helpful.

One consideration to make the error easier to search could be to mention the module without using it:
```suggestion
        MSG = "Prefer Rails built-in `t` function from the I18n Module over `ts`: the latter is not actually translatable. For more information, refer to https://github.com/otwcode/otwarchive/wiki/Internationalization-(i18n)-Standards."
```

But I don't really like it.